rfcs/detecting_sensitive_textual_content/20230309-implementation_plan_sensitive_terms_list.md

```diff
@@ -0,0 +1,158 @@
+      example, their
+      [thumbnail policy](https://support.google.com/youtube/answer/9229980?hl=en&ref_topic=9282679#zippy=%2Cage-restricted-thumbnails-and-thumbnail-removal)
+      includes some information on types of sexual and violent content that
+      either can't be used or can be used behind a flag comparable to ours.
+      - Additionally, see the [#this-will-not-be-perfect](#this-wont-be-perfect)
+        section for additional
+
+## Implementation Plan
+
+1. Ping a WordPress GitHub account maintainer to request that they create the
+   `WordPress/openverse-sensitive-terms` repository.
+2. Initialize the repository with a `README.md` and a blank
+   `sensitive-terms.txt` file.
+3. Create a PR which initializes the sensitive terms list by copying all terms
+   in the "List of Dirty, Naughty, Obscene, and Otherwise Bad Words" repository.
+   _I am not including the link here to prevent accidental viewing of the
+   repository, but feel free to find it within the GitHub organization
+   "LDNOOBW"._
+4. Identify willing reviewers from the `@WordPress/openverse-maintainers` group
+   who are comfortable with looking at the list and reviewing the PR, looking
+   for terms which should be removed or missing terms which should be added.
+5. Merge the PR, and now the list is available for use in Openverse projects.
+6. Publish a "Make" post about the creation of the list and its purpose.
+
+## This Will Not Be Perfect
+
+By default, we accept that this list and its usage will not be perfect. At times
+we may accidentally include terms which inadvertently cause harm to groups or
+individuals. We may also accidentally expose users to undesired content in the
+implementation of this list. Despite the potential for these transgressions this
+work will always be based on the following assumptions:
+
+1. Safety first. Given the choice, we would prefer to incorrectly restrict
+   non-sensitive content rather than accidentally show sensitive content.
+2. Avoid censorship. This list will not be used to block or deny access to
+   content, rather to make the viewing of content opt-in. The Openverse project
+   will use other means to explicitly block or remove content. Removing
+   sensitive content can, in many cases, be its own form of harm. A quote From
+   the research paper "On the Dangers of Stochastic Parrots: Can Language Models
+   Be Too Big? 🦜"[1] illustrates this well:
+
+   “If we filter out the discourse of marginalized populations, we fail to
+   provide training data that reclaims slurs and otherwise describes
+   marginalized identities in a positive light”
+
+3. Encourage feedback. We will request and monitor community feedback on the
+   terms in this list (without inadvertently exposing anyone to the list) to
+   make sure it meets the expectations of our users.
+
+## Open questions for reviewers
+
+- Within the Openverse maintainers; who has to view this list? Can anyone opt
+  out?
```

Comment:
I think we should be able to opt out.

I'm also comfortable volunteering to review the list of words.

However, I will ask: what is the intention behind reviewing the list of words? Are we reviewing for format or to review the actual words themselves and make recommendations on their inclusion/exclusion?

Reply:
> Are we reviewing for format or to review the actual words themselves and make recommendations on their inclusion/exclusion?

I think both of these would be important but primarily reviewing the actual words themselves.